pipeline/config.go

```diff
@@ -24,14 +24,20 @@ type Config []operator.Config
 // BuildOperators builds the operators from the list of configs into operators
 func (c Config) BuildOperators(bc operator.BuildContext) ([]operator.Operator, error) {
 	operators := make([]operator.Operator, 0, len(c))
-	for i, builder := range c {
-		nbc := getBuildContextWithDefaultOutput(c, i, bc)
-		op, err := builder.Build(nbc)
+	pluginMap := make(map[string]string)
+	for _, builder := range c {
+		op, err := builder.Build(bc)
 		if err != nil {
 			return nil, err
 		}
+
+		if len(op) > 1 {
```

Comment:
the best Ideas I can think of off the top of my head other than this would be maybe trying to assert that it is Plugin and if that works then 'do work', or adding it to the entire interface which would just return false for all other ops. I was trying to avoid the latter because I've already added two new signatures to the `Operator` interface.

Reply:
ok added it to the builder interface so that we can determine when there are multiple outputs through the new `BuildsMultipleOps` method.